### DISCUSSION TITLE
wezterm doesn't display chinese normally

### DISCUSSION THREAD

**Discussion by r-xiumu** [wezterm/wezterm#2846] [Q&A]
I set default_prog to git bash.
When I input `ls` in wezterm, it display messy code instead of chinese  .  

![image](https://user-images.githubusercontent.com/91208685/206244414-d6e11b37-9b2b-4148-aecf-c800dcdc98c4.png)
But when I input `ls` in git bash, it display chinese normally.  

![image](https://user-images.githubusercontent.com/91208685/206245814-e9c5195b-5b28-4a46-98a7-c893236ef025.png)

after I search answer in issues, i found 2 issue:
[#299](https://github.com/wez/wezterm/issues/299)
I set `font = wezterm.font_with_fallback {'SimHei'}` , but not work.

[#250](https://github.com/wez/wezterm/issues/250)
This is too long, I don't understand...

What I can do to resolve this question fast?

**Comment by wez** [comment]
Make sure that LANG and locale related things are set correctly in your shell.
https://wezfurlong.org/wezterm/faq.html#lang-and-locale
wezterm only supports UTF-8 based encoding. If you have selected big5 or other encoding, then it will not be rendered correctly.

**Reply by r-xiumu** [reply]
Thanks very much.
Chinese language display normally by setting `export LANG=en_US.UTF-8` in ~/.bashrc.
